src/main/java/org/elasticsearch/index/engine/CommitStats.java

```diff
@@ -35,24 +36,20 @@
 
     private Map<String, String> userData;
     private long generation;
+    private int numDocs;
```

Comment:
DocStats are on the indexreader which might have modifications compared tot he commit point. this is the num docs at the point we committed